### PR TITLE
Feature/pubsub integrasi labkes

### PR DESCRIPTION
**Pull request by sandisunandar99**
## Overview 

- Integrasi Lapor mandiri

_note:_
_module integrasi labkes di comment dlu sengaja_

### CODE REVIEW

**Comment by firmanJS** [comment]
base na ti development iyeu mang  @sandisunandar99 ?

**Comment by sandisunandar99** [comment]
> base na ti development iyeu mang @sandisunandar99 ?

asa na mah enya ti development,, kunaon kitu ??

**Comment by firmanJS** [comment]
> > base na ti development iyeu mang @sandisunandar99 ?
> 
> asa na mah enya ti development,, kunaon kitu ??

fitur cache pan aya nu di development mang, api mah ges siap tapi di fe na can siap

**Comment by sandisunandar99** [comment]
> > > base na ti development iyeu mang @sandisunandar99 ?
> > 
> > 
> > asa na mah enya ti development,, kunaon kitu ??
> 
> fitur cache pan aya nu di development mang, api mah ges siap tapi di fe na can siap, karena belum ada desain redaksi na

oh ,, sigana teu asup mang iey mah di branch integrasi langung ,, cek we commitan na ,, aya nu asup teu ? mun aya ieu di close hela PR na

File: helpers/reports/filter.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+validateQuery = (key, query, user) => {
+  if(query.date && query.address_district_code) {
+    key = `daily-reports-${user.username}-${query.date}-${query.address_district_code}`
+  } else if(query.address_district_code) {
+    key = `daily-reports-${user.username}-${query.address_district_code}`
+  } else if(query.date) {
+    key = `daily-reports-${user.username}-${query.date}`
+  } else {
+    key = `daily-reports-${user.username}`
+  }
```
Comment (firmanJS) [inline helpers/reports/filter.js:12]: aya mang iyeu asup :crying_cat_face: kabawa :laughing:

Comment (sandisunandar99) [reply helpers/reports/filter.js:12]: wah nana aya di branch pubsub nya

Comment (firmanJS) [reply helpers/reports/filter.js:12]: berarti nt nge pull base branch development mang

Comment (sandisunandar99) [reply helpers/reports/filter.js:12]: jadi kumaha nya,, di cerry pick wae kitu,, tapi da loba commitan na,, wkwkwk :laughing:

Comment (sandisunandar99) [reply helpers/reports/filter.js:12]: coba ke mang di daily,,, rada di mention nu redaksi tadi.. biar segera naik prod,, meh teu numpuk tuing

Comment (firmanJS) [reply helpers/reports/filter.js:12]: arek di chery pick hegae, atawa ngoding ulang di branch lain base master asupkeun kode nu rek di push ka master na wkwkwk, enya mun tetep hayang up kabeh nu didieu kudu follow up ka si erlangga redaksi ker cache

Comment (sandisunandar99) [reply helpers/reports/filter.js:12]: edaan kitu ,, ngoding ulang :rofl:

Comment (firmanJS) [reply helpers/reports/filter.js:12]: jadi kumaha atuh mang wkwkkw

**Pull request closed without merging**